src/tools/rbd/action/MirrorPool.cc

```diff
@@ -1430,18 +1471,28 @@ int execute_info(const po::variables_map &vm,
     std::cout << "Mode: " << mirror_mode_desc << std::endl;
   }
 
-  if (mirror_mode != RBD_MIRROR_MODE_DISABLED && namespace_name.empty()) {
+  if (mirror_mode != RBD_MIRROR_MODE_DISABLED) {
+    if (namespace_name.empty()) {
+      if (formatter != nullptr) {
+        formatter->dump_string("site_name", site_name);
+      } else {
+	std::cout << "Site Name: " << site_name << std::endl;
+      }
+    }
     if (formatter != nullptr) {
-      formatter->dump_string("site_name", site_name);
+      formatter->dump_string("mirror_uuid", mirror_uuid);
+      formatter->dump_string("remote_namespace", remote_namespace);
     } else {
-      std::cout << "Site Name: " << site_name << std::endl
-                << std::endl;
+      std::cout << "Mirror UUID: " << mirror_uuid << std::endl;
```

Comment:
Noting that adding mirror UUID to `rbd mirror pool info` output is an unrelated change -- it would be good to split it into a separate commit.